WordPressKit/MediaServiceRemoteXMLRPC.m

```diff
@@ -53,14 +55,17 @@ - (void)getMediaLibraryStartOffset:(NSUInteger)offset
                          return;
                      }
                      NSArray *pageMedia = [self remoteMediaFromXMLRPCArray:responseObject];
+                     if(pageLoad) {
```

Comment:
The REST version of this only calls `pageLoad` if media has a > 0 item count, but it seems like this version calls it every time. I'm not sure what the 'correct' option would be, but I think these two should be consistent.